pkg/policy/policy_test.go

```diff
@@ -518,6 +534,116 @@ func TestPolicies_Eval(t *testing.T) {
 			policies:           make(map[string]string),
 			expectedError:      policy.PoliciesNotFoundError,
 		},
+		{
+			name:               "using a custom rule - check passed",
+			resource:           simpleNginxPod,
+			useBuiltInPolicies: false,
+			policies: map[string]string{
+				"policy.uses_image_tag_latest.kinds": "Pod",
+				"policy.uses_image_tag_latest.rego": `
+   package trivyoperator.policy.k8s.custom
+   __rego_metadata__ := {
+        "id": "CUSTOMCHECK",
+        "title": "custom check title",
+        "severity": "LOW",
+        "type": "Kubernetes Security Check",
+        "description": "custom check description",
+    }
+
+   failHostNetwork {
+      1 == 0
+   }
+
+   deny[res] {
+        failHostNetwork
+		res := {
+				"msg": "the check should always pass",
+			}
+   }`,
```

Comment:
The rules can be shortened:
```suggestion
   deny[{"msg": "the check should always pass"}]`,
```

Reply:
isn't this suggestion negates the check logic? before it has a false statement, now it doesn't

Reply:
this place is refactored.